contracts/schemes/ExternalLocking4Reputation.sol

```diff
@@ -33,36 +37,45 @@ contract ExternalLocking4Reputation is Locking4Reputation, Ownable {
     function initialize(
         Avatar _avatar,
         uint _reputationReward,
-        uint _lockingStartTime,
-        uint _lockingEndTime,
+        uint _claimingStartTime,
+        uint _claimingEndTime,
         uint _redeemEnableTime,
         address _externalLockingContract,
         string _getBalanceFuncSignature)
     external
     onlyOwner
     {
-        require(_lockingEndTime > _lockingStartTime, "_lockingEndTime should be greater than _lockingStartTime");
+        require(_claimingEndTime > _claimingStartTime, "_claimingEndTime should be greater than _claimingStartTime");
         externalLockingContract = _externalLockingContract;
         getBalanceFuncSignature = _getBalanceFuncSignature;
         super._initialize(
         _avatar,
         _reputationReward,
-        _lockingStartTime,
-        _lockingEndTime,
+        _claimingStartTime,
+        _claimingEndTime,
         _redeemEnableTime,
         1);
     }
 
     /**
-     * @dev lock function
-     * @return lockingId
+     * @dev claim function
+     * @param _beneficiary the beneficiary address to claim for
+     *        if _beneficiary == 0 the claim will be for the msg.sender.
+     * @return claimId
      */
-    function lock() public returns(bytes32) {
+    function claim(address _beneficiary) public returns(bytes32) {
```

Comment:
Can we call "_beneficiary"  "_claimer"?

Reply:
But it is not the claimer, as one can claim for someone else, who was registered.

Reply:
@leviadam 

msg.sender in this case is claiming on behalf of the claimant (here `_beneficiary`), who "registered"  (poor word IMHO) to allow someone else (here msg.sender) to "claim" on their behalf. 

`msg.sender` is not here claiming anything, rather is executing a claim an behalf of someone else.